src/promptflow/promptflow/exceptions.py

```diff
@@ -47,6 +31,7 @@ class ErrorTarget(str, Enum):
     SERVING_APP = "ServingApp"
     FLOW_INVOKER = "FlowInvoker"
     FUNCTION_PATH = "FunctionPath"
+    SDK = "SDK"
```

Comment:
we already have CONTROL_PLANE_SDK, is that used elsewhere? maybe rename that to SDK?

Reply:
There are many places where **CONTROL_PLANE_SDK** is used, so should I remove the **SDK** and use **CONTROL_PLANE_SDK** directly?
![Uploading image.png…]()

Reply:
That should also be fine. But pls confirm with T&I guys on their preference. How will this be used in dashboard?

Reply:
We have already communicated, and we should mainly use error category.